src/Type/Regex/RegexExpressionHelper.php

```diff
@@ -88,13 +91,24 @@ public function getPatternModifiers(string $pattern): ?string
 
 	public function removeDelimitersAndModifiers(string $pattern): string
 	{
+		$pattern = ltrim($pattern);
```

Comment:
@staabm in `getPatternDelimiter` we do `ltrim` but here it was missing, but tests are not failing - do you know why? However, I belive `ltrim` should be here as we strip the start delimiter using `substr($v, 1, ...`.